selfdrive/ui/qt/maps/map.cc

```diff
@@ -139,20 +139,25 @@ void MapWindow::updateState(const UIState &s) {
   const SubMaster &sm = *(s.sm);
   update();
 
+  // show map on rising edge of navigate on openpilot
   if (sm.updated("modelV2")) {
-    // set path color on change, and show map on rising edge of navigate on openpilot
     bool nav_enabled = sm["modelV2"].getModelV2().getNavEnabled();
-    if (nav_enabled != uiState()->scene.navigate_on_openpilot) {
-      if (loaded_once) {
-        m_map->setPaintProperty("navLayer", "line-color", getNavPathColor(nav_enabled));
-      }
-      if (nav_enabled) {
-        emit requestVisible(true);
-      }
+    if (nav_enabled && !uiState()->scene.navigate_on_openpilot) {
```

Comment:
shouldn't this be the same check as the color?

Reply:
I think the interaction would be more confusing that way (ie. the map won't appear until you first engage, then you can hide it again when you disengage)

Reply:
yeah, I think this is confusing since you don’t care if the model is using the map when you’re not engaged

Reply:
changed! it will let you hide map and change color to blue when disengaged, but nav features are being used

Reply:
yes

Reply:
I still like always showing the map, but we can see how it feels in the car